src/lib/PointCanvas.ts

```diff
@@ -214,6 +214,19 @@ export class PointCanvas {
         this.controls.update();
     };
 
+    lockRotation(ndim: number) {
+        if (ndim == 2) {
+            const cameraPosition = new ViewerState().cameraPosition;
+            this.camera.position.set(cameraPosition[0], cameraPosition[1], cameraPosition[2]);
+            this.controls.enableRotate = false;
```

Comment:
It seems like the autoRotate button (the DNA button in the bottom right) still has an effect after this. Ideally we would disable that too because it's somewhat useless in this case.

Reply:
Good catch, thanks!